docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplateBase.java

```diff
@@ -110,16 +111,32 @@ public DockerTemplateBase(String image,
 
     protected Object readResolve() {
         if (volumesFrom != null) {
```

Comment:
So you can just write `will be fixed next time` and merge without any doubt. And next time don't write duplicated code.